### PR TITLE
Update Oculus to API 1.44

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview

Update Oculus to API 1.44, now includes Oculus Quest Hand Tracking API support (awaiting XRTK hands support)

## Changes:

- Updated: Oculus API to include new headers and wrappers
- Updated: Manifest Generator to enable hands support
- Fixes: #38

### CODE REVIEW

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs [hunk 1]
```diff
@@ -70,6 +70,27 @@ public static void GenerateManifestForSubmission()
 				Debug.LogWarning("Manifest error: unable to locate headset DoF mode");
 			}
 
+			int handTrackingTextIndex = manifestText.IndexOf("<!-- Request the headset handtracking mode -->");
+			if (handTrackingTextIndex != -1)
+			{
+
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs]: I'm really confused about this class. It it part of the xrtk or does it belong to facebook?

Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs]: That is Oculus "stuff", trying to keep aligned to how they build their Manifest

File: XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs [hunk 2]
```diff
@@ -25,60 +25,81 @@
...
+                Debug.LogWarning("Manifest error: unable to locate headset DoF mode");
+            }
+
+            int handTrackingTextIndex = manifestText.IndexOf("<!-- Request the headset handtracking mode -->");
+            if (handTrackingTextIndex != -1)
+            {
+
+                    bool handTrackingEntryNeeded = true; // (targetHandTrackingSupport != OVRProjectConfig.HandTrackingSupport.ControllersOnly);
+                    bool handTrackingRequired = false; // (targetHandTrackingSupport == OVRProjectConfig.HandTrackingSupport.HandsOnly);
+                    if (handTrackingEntryNeeded)
```
Comment (StephenHodgson) [inline XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs]: handTrackingEntryNeeded is always true?

Comment (SimonDarksideJ) [reply XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs]: For now until we have a Platform system with Hands configuration to alter it
It's set to enable projects to be able to work with EITHER hands or controllers

**Pull request merged**